flink-cdc-connect/flink-cdc-source-connectors/flink-connector-mysql-cdc/src/test/java/org/apache/flink/cdc/connectors/mysql/testutils/UniqueDatabase.java

```diff
@@ -58,10 +62,19 @@ public class UniqueDatabase {
 
     public UniqueDatabase(
             MySqlContainer container, String databaseName, String username, String password) {
+        this(container, databaseName, getIdentifier(), username, password);
+    }
+
+    public UniqueDatabase(
+            MySqlContainer container,
+            String databaseName,
+            String username,
+            String password,
+            boolean withIdentifier) {
         this(
                 container,
                 databaseName,
-                Integer.toUnsignedString(new Random().nextInt(), 36),
+                withIdentifier ? getIdentifier() : StringUtils.EMPTY,
```

Comment:
The identifier suffix is meant to avoid test case conflicts since multiple cases might run simultaneously on a single MySQL database instance. It should be fine now since there's only one test case using this class, but might cause trouble in the future if more testcases are added.